libs/core/src/lib/multi-input/multi-input.component.ts

```diff
@@ -579,6 +609,26 @@ export class MultiInputComponent
         this.openChangeHandle(!this.open);
     }
 
+    /** @hidden */
+    private _updateSearchAnnoucementText(): void {
+        if (this.searchTerm) {
+            // create search suggestion message with count.
+            this._currentSearchSuggestionAnnoucementMessage =
+                this.displayedValues?.length +
+                ' ' +
+                this.searchSuggestionMessage +
+                (this.displayedValues?.length > 0 && !this.mobile ? ',' + this.searchSuggestionNavigateMessage : '');
```

Comment:
Maybe better to handle this with replacements of some placeholders inside one string, instead of composing it from multiple separate strings? This might be handy if order of the words and counters isn't matching this pattern.

Reply:
fixed